### ISSUE TITLE
Horizontal scroll bar in message chops of bottom of text

### ISSUE THREAD

**Issue by ashb** [Foundry376/Mailspring#1762]
##### What operating system are you using?

Arch Linux

##### What version of Mailspring are you using?

1.7.2

##### Is the issue reproducible with a particular attachment, message, signature, etc?

The last line of the message is cut off when a horizontal scroll bar is shown.

![image](https://user-images.githubusercontent.com/34150/72091836-016f1200-3309-11ea-8c08-bfeb84b0c251.png)

I've also noticed this with Github PR comment emails where the "View in GitHub" is not visible)

The HTML email content that is producing this is:

```
<o:p =
class=3D""></o:p></span></div><div style=3D"margin: 0in 0in 0.0001pt; =
font-size: 12pt; font-family: Calibri, sans-serif;" class=3D""><span =
style=3D"font-size: 11pt;" class=3D"">&nbsp;</span></div><pre =
style=3D"margin: 0in 0in 3pt; font-size: 10pt; font-family: =
&quot;Courier New&quot;;" class=3D""><span style=3D"font-size: 9pt; =
color: rgb(29, 28, 29);" class=3D"">airflow:<br class=3D"">&nbsp; =
config:<br class=3D"">&nbsp;&nbsp;&nbsp; AIRFLOW__CORE__REMOTE_LOGGING: =
"True"<br class=3D"">&nbsp;&nbsp;&nbsp; # HTTP_PROXY: "<a =
href=3D"https://slack-redir.net/link?url=3Dhttp%3A%2F%2Fproxy.mycompany.co=
m%3A123" target=3D"_blank" =
title=3D"https://slack-redir.net/link?url=3Dhttp%3A%2F%2Fproxy.mycompany.c=
om%3A123" style=3D"color: rgb(5, 99, 193);" =
class=3D"">http://proxy.mycompany.com:123</a>"<br =
class=3D"">&nbsp;&nbsp;&nbsp; AIRFLOW__ELASTICSEARCH__LOG_ID_TEMPLATE: =
"{{dag_id}}-{{task_id}}-{{execution_date}}-{{try_number}}"<br =
class=3D"">&nbsp;&nbsp;&nbsp; AIRFLOW__ELASTICSEARCH__END_OF_LOG_MARK: =
"end_of_log"<br class=3D"">&nbsp;&nbsp;&nbsp; =
AIRFLOW__ELASTICSEARCH__WRITE_STDOUT: "True"<br =
class=3D"">&nbsp;&nbsp;&nbsp; AIRFLOW__ELASTICSEARCH__JSON_FORMAT: =
"True"<br class=3D"">&nbsp;&nbsp;&nbsp; =
AIRFLOW__ELASTICSEARCH__JSON_FIELDS: "asctime, filename, lineno, =
levelname, message"<o:p class=3D""></o:p></span></pre><div =
style=3D"margin: 0in 0in 0.0001pt; font-size: 12pt; font-family: =
Calibri, sans-serif;" class=3D""><br class=3D""></div></body></html>=
```

**Comment by ashb** [comment]
Playing with the debug inspector the problem is the last element in the message contains a margin-bottom of only 3px, which messes up things.

**Comment by bengotow** [comment]
Hey folks, thanks for reporting this. Happy to say this has been resolved in 1.7.6 which shipped this afternoon. 🚀  The app should auto-update, but if it doesn't you can also grab the new release from GitHub's Releases tab or from https://getmailspring.com/download. Thanks for your patience and help tracking this down, enjoy the update!
